deploy/install.go

```diff
@@ -97,8 +97,8 @@ const (
 	certConfigManifest           = "signed-cert.yaml"
 	bundleTarPath                = "dist/karavi-airgap-install.tar.gz"
 	karavictl                    = "karavictl"
-	sidecarImageTar              = "sidecar-proxy-latest.tar"
-	sidecarDockerImage           = "sidecar-proxy:latest"
+	sidecarImageTar              = "sidecar-proxy-1.0.0.tar"
```

Comment:
Can we pull this out of code so we don't need to change it every time there is a new release?

Reply:
Done